transport/conn.go

```diff
@@ -427,7 +433,12 @@ func (c *Conn) Shard() int {
 // Close closes connection and terminates reader and writer go routines.
 func (c *Conn) Close() {
 	log.Printf("%s closing", c)
-	_ = c.conn.Close()
+	err := c.conn.Close()
+	if err != nil {
+		// Error happens when either conn or file descriptor is nil or there
+		// has been error when closing file descriptor.
+		log.Printf("%s conn.Close() error", c)
```

Comment:
Should we add some go func() that once in a while tries again to close connections on which conn.Close() errors have been recorded?
Worst case now we have open file descriptor which seems pretty bad.

Reply:
I don't think it works like that. 
log.Fatal seems to be reasonable at this point

Reply:
Based on my quick internet sweep I haven't found projects that actually care about error on `conn.Close()` so I think regular `log.Printf()` will be more than enough.

> Should we add some go func() that once in a while tries again to close connections on which conn.Close() errors have been recorded?

This isn't that big of a deal to do that now I guess.

Reply:
agree